app/components/SCForm.js

```diff
@@ -82,6 +90,7 @@ class SCForm extends Component {
       this.TcombType = transform(schema);
       this.initialValues = initialValues;
       this.options = options;
+      this.setState({ message: ''});
```

Comment:
Seems like this is unnecessary b/c that's the default value set in the constructor?